docs/topics/testing/tools.txt

```diff
@@ -606,6 +606,16 @@ features like:
 then you should use :class:`~django.test.TransactionTestCase` or
 :class:`~django.test.TestCase` instead.
 
+.. attribute:: SimpleTestCase.allow_database_queries
+
+    .. versionadded:: 1.9
+
+    :class:`~SimpleTestCase` will prevent database queries by raising
```

Comment:
SimpleTestCase disallows database queries by default. This helps to avoid executing write queries which will affect other tests since each SimpleTestCase test isn't run under a transaction. If you aren't concerned about this problem, you can disable...